src/incubator/TextField/types.ts

```diff
@@ -86,6 +86,7 @@ export interface FloatingPlaceholderProps {
   floatOnFocus?: boolean;
   validationMessagePosition?: ValidationMessagePosition;
   extraOffset?: number;
+  defaultValue?: TextInputProps['defaultValue'];
```

Comment:
'TextInputProps' are the OLD TextField's prop... why use them in the incubator?

Reply:
No, it's RN's props

Reply:
Ok...